mozci/task.py

```diff
@@ -402,6 +402,18 @@ def classifications(self):
             and any(result.group == self.name and not result.ok for result in t.results)
         ]
 
+    @property
+    def durations(self):
+        return [t.duration for t in self.tasks]
```

Comment:
Yeah, this one isn't right since we need to sum the durations of the "group" that this `GroupSummary` object corresponds to, not the tasks. That's what we'll need the AD query for.